samples/matter/thermostat/prj.conf

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# This sample uses Kconfig.defaults to set options common for all
+# samples. This file should contain only options specific for this sample
+# or overrides of default values.
+
+# Enable CHIP
+CONFIG_CHIP=y
+CONFIG_CHIP_ENABLE_PAIRING_AUTOSTART=y
+CONFIG_CHIP_PROJECT_CONFIG="src/chip_project_config.h"
+# 32782 == 0x800e (example Product ID added temporaly,
+# but it must be changed with proper PID from the list:
+# https://github.com/project-chip/connectedhomeip/blob/482e6fd03196a6de45465a90003947ef4b86e0b1/docs/examples/discussion/PID_allocation_for_example_apps.md)
+CONFIG_CHIP_DEVICE_PRODUCT_ID=32782
+CONFIG_STD_CPP14=y
+
+# Add support for LEDs and buttons on Nordic development kits
+CONFIG_DK_LIBRARY=y
+
+# Bluetooth Low Energy configuration
+CONFIG_BT_DEVICE_NAME="MatterThermo"
+
+# Other settings
+CONFIG_THREAD_NAME=y
+CONFIG_MPU_STACK_GUARD=y
+CONFIG_RESET_ON_FATAL_ERROR=n
+CONFIG_CHIP_LIB_SHELL=y
+
+# Disable NFC commissioning
+CONFIG_CHIP_NFC_COMMISSIONING=y
+
+# Reduce application size
+CONFIG_USE_SEGGER_RTT=n
+
+
+
+CONFIG_TEMP_NRF5=y
```

Comment:
1. Too many empty lines
2. Is this config really used now? I guess not. Anyway, even if we support real sensor measurement on board, this will not work on nRF53 as far as I know.